src/idex.ts

```diff
@@ -21,7 +20,7 @@ export class IdentityResolver {
   url: string
   timeout: number
   requestedAttributes: string[]
-  tuples: [string, string][]
+  query: QueryBuilder // ensure to copy this before mutating
```

Comment:
Find the comment really helpful. But maybe provide more details here? Something like "Be careful, this object is mutable. In cases where temporary changes are needed - e.g. adding parameters that are only valid for one single call - ensure to copy it".